### PR TITLE
TPI: narrower banner

### PR DESCRIPTION
**Pull request by casperdcl**
For https://github.com/iterative/terraform-provider-iterative/pull/433

Mainly to reduce height.

Before

After

### CODE REVIEW

**Review by 0x2b3bfa0 — APPROVED** [review]
Not a fan of the design, but the new version is definitely more compact and better than a placeholder.

**Comment by 0x2b3bfa0** [comment]
## SVG
tpi

**Comment by shcheklein** [comment]
Should we use https://www.stefanjudis.com/notes/how-to-define-dark-light-mode-images-in-github-markdown/ ?

Also, we can ask @yalozhkin and @arcticbear to improve it if we have some ideas.

**Comment by shcheklein** [comment]
Screen Shot 2022-03-10 at 2 41 36 PM

This is how it looks like for me (font is not readable).

**Comment by 0x2b3bfa0** [comment]
I didn't want to fix this until the layout was finished because it makes modifications more difficult, but here it is.
157766255-8447e0f2-03b1-492e-b2e8-6af9a8a2d6a2
157766255-8447e0f2-03b1-492e-b2e8-6af9a8a2d6a2

**Comment by 0x2b3bfa0** [comment]
> _Should we use https://www.stefanjudis.com/notes/how-to-define-dark-light-mode-images-in-github-markdown?_

Wow! I didn't know this trick. ❤️

**Comment by 0x2b3bfa0** [comment]
github com_iterative_static_pull_8 (1)
github com_iterative_static_pull_8 (2)

**Comment by 0x2b3bfa0** [comment]
(Took the liberty of modifying logo colors to have roughly the same contrast on light & dark modes)

**Comment by 0x2b3bfa0** [comment]
Monochrome might look better in dark mode, but I'll leave that to professionals. Enough dilettante–level “design” for today.

**Comment by yalozhkin** [comment]
@0x2b3bfa0 where can I get the original Terraform logo SVG? When I try to open the logo from the thread, it looks broken.

**Comment by 0x2b3bfa0** [comment]
https://www.hashicorp.com/brand#terraform

File: img/cml/banner-tpi.svg [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<svg width="792" height="190" viewBox="0 0 792 190" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (casperdcl) [inline img/cml/banner-tpi.svg:1]: could we do 1715x322 @arcticbear?

Comment (arcticbear) [reply img/cml/banner-tpi.svg:1]: Sure, here is an updated one.
terraform-iterative-dark-bg-3

**Pull request merged**